PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,10 +16,10 @@ Fixes # (issue number)
 Please delete options that are not relevant.
 -->
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
+-   [ ] Bug fix (non-breaking change which fixes an issue)
+-   [ ] New feature (non-breaking change which adds functionality)
+-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+-   [ ] This change requires a documentation update
```

Comment:
Why are there extra spaces here?

Reply:
I didn't touch the file, prettier certainly did